public/login.html

```diff
@@ -6,7 +6,9 @@
 </head>
 
 <body> 
+    <div class="newpeople">
```

Comment:
I would suggest indenting your code properly. If you need a refresher on indenting, here one is: https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/spacing-indentation-html.html